samples/snippets/classification_boosted_tree_model_test.py

```diff
@@ -39,4 +39,27 @@ def test_boosted_tree_model(random_model_id: str) -> None:
     )
     del input_data["functional_weight"]
     # [END bigquery_dataframes_bqml_boosted_tree_prepare]
+    # [START bigquery_dataframes_bqml_boosted_tree_create]
+    # from sklearn.ensemble import GradientBoostingClassifier
+    import bigframes.ml.linear_model
+
+    # input_data is defined in an earlier step.
+    training_data = input_data[input_data["dataframe"] == "training"]
+    X = training_data.drop(columns=["income_bracket", "dataframe"])
+    y = training_data["income_bracket"]
+
+    # create and train the model
+    census_model = bigframes.ml.linear_model.LogisticRegression(
+        # model_type="BOOSTED_TREE_CLASSIFIER",
+        # booster_type="gbtree",
+        max_iterations=50,
+    )
```

Comment:
I don't think we should be doing `LogisticRegression` here. In the SQL we do use `model_type='BOOSTED_TREE_CLASSIFIER'`, but in BigQuery DataFrames we normally use separate Python classes to represent the different model types.

A few ways to discover which class to use:

* Search our [code for `BOOSTED_TREE_CLASSIFIER`](https://github.com/search?q=repo%3Agoogleapis%2Fpython-bigquery-dataframes%20BOOSTED_TREE_CLASSIFIER&type=code)
* Google search for [boosted trees BigFrames](https://www.google.com/search?q=boosted+trees+bigframes)

These should give you some strong hints as to which class to use instead.

Reply:
Copying this from an internal comment I made for visibility:

Just like scikit-learn, it's one of the "ensemble" methods: [https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.ml.ensemble](https://www.google.com/url?q=https://cloud.google.com/python/docs/reference/bigframes/latest/bigframes.ml.ensemble&sa=D&source=docs&ust=1731968708166884&usg=AOvVaw2y30ZNkl2x1T5ip52wSzNr)

Normally we try to use the scikit-learn class names too, but I think we may have added this class before GradientBoostingClassifier as in scikit-learn